### PR TITLE
Updating language compatibility for ASM products

### PR DESCRIPTION
**Pull request by sourabhkatti**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by aliciascott — APPROVED** [review]
approved for docs

File: layouts/shortcodes/asm-libraries-capabilities.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ The following ASM capabilities are supported relative to each language's tracing
 | ASM capability                   | Java | .NET | Node.js | Python | Go | Ruby | PHP |
 | -------------------------------- | ----------------------------------|----------------------------|----------------------------|----------------------------|----------------------------|----------------------------|----------------------------|
 | Threat Detection| 1.8.0 | 2.23.0 | 3.13.1 | 1.9.0   | 1.47.0  | 1.9.0| 0.84.0 |
+| API Security | 1.31.0 |  |  |  |  |  |  |
```
Comment (Hellzy) [inline layouts/shortcodes/asm-libraries-capabilities.md:7]:
```suggestion
| API Security | 1.31.0 |  |  |  | 1.59.0 |  |  |
```

File: content/en/security/application_security/api-inventory/_index.md [hunk 2]
```diff
@@ -31,6 +31,10 @@ The following library versions are compatible with API Security Inventory. [Remo
 |Technology|Minimum version| Support for sensitive data scanning |
 |----------|----------|----------|
 |Python    | v2.1.6   | Requests and responses |
+|Java      | v1.31.0  | Requests only |
+|Php      | v0.98.0  | Requests and responses |
```
Comment (smola) [inline content/en/security/application_security/api-inventory/_index.md:35]:
```suggestion
|PHP      | v0.98.0  | Requests and responses |
```
For consistency

File: content/en/security/application_security/api-inventory/_index.md [hunk 3]
```diff
@@ -31,6 +31,10 @@ The following library versions are compatible with API Security Inventory. [Remo
...
+|.NET Core | v2.42.0  | Requests and responses |
+|.NET Fx   | v2.47.0  | Requests and responses |
```
Comment (lloeki) [inline content/en/security/application_security/api-inventory/_index.md:37]:
```suggestion
|.NET Fx   | v2.47.0  | Requests and responses |
|Ruby      | v1.15.0  | Requests only |
```

File: layouts/shortcodes/asm-libraries-capabilities.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ The following ASM capabilities are supported relative to each language's tracing
 | ASM capability                   | Java | .NET | Node.js | Python | Go | Ruby | PHP |
 | -------------------------------- | ----------------------------------|----------------------------|----------------------------|----------------------------|----------------------------|----------------------------|----------------------------|
 | Threat Detection| 1.8.0 | 2.23.0 | 3.13.1 | 1.9.0   | 1.47.0  | 1.9.0| 0.84.0 |
+| API Security | 1.31.0 | 2.42.0 |  |  | 1.59.0 |  | 0.98.0 |
```
Comment (lloeki) [inline layouts/shortcodes/asm-libraries-capabilities.md:7]:
```suggestion
| API Security | 1.31.0 | 2.42.0 |  |  | 1.59.0 | 1.15.0 | 0.98.0 |
```

**Review by lloeki — COMMENTED** [review]
Suggested addition of Ruby version to tables

File: content/en/security/application_security/api-inventory/_index.md [hunk 5]
```diff
@@ -31,7 +31,12 @@ The following library versions are compatible with API Security Inventory. [Remo
 |Technology|Minimum version| Support for sensitive data scanning |
 |----------|----------|----------|
 |Python    | v2.1.6   | Requests and responses |
-|Golang    | v1.59.0  | Requests |
+|Java      | v1.31.0  | Requests only |
+|PHP      | v0.98.0  | Requests and responses |
+|.NET Core | v2.42.0  | Requests and responses |
+|.NET Fx   | v2.47.0  | Requests and responses |
+|Ruby      | v1.15.0  | Requests only |
+|Golang    | v1.59.0  | Requests only |
```
Comment (iunanua) [inline content/en/security/application_security/api-inventory/_index.md:40]: We maintain three release lines, v5, v4 and v3 depending on the nodejs runtime so

```suggestion
|Node.js | v5.6.0, v4.30.0, v3.51.0 | Requests and responses |
```

File: layouts/shortcodes/asm-libraries-capabilities.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ The following ASM capabilities are supported relative to each language's tracing
 | ASM capability                   | Java | .NET | Node.js | Python | Go | Ruby | PHP |
 | -------------------------------- | ----------------------------------|----------------------------|----------------------------|----------------------------|----------------------------|----------------------------|----------------------------|
 | Threat Detection| 1.8.0 | 2.23.0 | 3.13.1 | 1.9.0   | 1.47.0  | 1.9.0| 0.84.0 |
+| API Security | 1.31.0 | 2.42.0 |  |  | 1.59.0 | 1.15.0 | 0.98.0 |
```
Comment (iunanua) [inline layouts/shortcodes/asm-libraries-capabilities.md:7]:
```suggestion
| API Security | 1.31.0 | 2.42.0 | 5.6.0, 4.30.0, 3.51.0 |  | 1.59.0 | 1.15.0 | 0.98.0 |
```

**Review by simon-id — APPROVED** [review]
nodejs lgtm

**Review by smola — APPROVED** [review]
LGTM.

**Comment by aliciascott** [comment]
Hi @sourabhkatti this looks good! I had to update the lines "The minimum tracer version to get all supported ASM capabilities for <library>" to the new versions required for API Security, also API Security was missing from Ruby so I added that, lastly I added "not supported" to the main compatibility page for Python for API Security. Please do another check before we merge :)

**Pull request merged**